agents_infra/agents_infra/utils/communication.py

```diff
@@ -0,0 +1,32 @@
+from logtools import maybe_log_message
+from token_manager import TokenManager
+
+
+class AgentCommunicator(object):
```

Comment:
Add a docstring describing the purpose and use of this class, its parameters, and its main responsibilities.